http/spring-boot-starter-web-cloudevents/README.md

```diff
@@ -0,0 +1,46 @@
+# HTTP Protocol Binding for Spring Boot Web (Reactive Stack)
+
+For Maven based projects, use the following to configure the CloudEvents Spring Boot Binding:
+
+```xml
+<dependency>
+    <groupId>io.cloudevents</groupId>
+    <artifactId>spring-boot-starter-web-cloudevents</artifactId>
+    <version>2.0.0-milestone1</version>
+</dependency>
+```
+
+## Receiving CloudEvents
+Below is a sample on how to read and write CloudEvents from a Post Request Headers and Body:
+
+```java
+    @PostMapping
+    public String recieveCloudEvent(@RequestHeader Map<String, String> headers, @RequestBody Object body) {
+      // Create a CloudEvent from Header and Body coming in the request
+      CloudEvent cloudEvent = CloudEventsHelper.parseFromRequest(headers, body);
+
+
+    }
+```
+
+
+## Sending CloudEvents
+
+
+Below is a sample on how to use the client to send a CloudEvent:
+
+```java
+  // Create the CloudEvent with the builder
+  final CloudEvent myCloudEvent = CloudEventBuilder.v03()
```

Comment:
Hiding the version  of the spec or the version of the builder (v03, V1, ....) is perhaps not a great idea. I would prefer that this version is part of the GAV consumed by the starter to make it available publicly to all the users.